pre_commit/error_handler.py

```diff
@@ -12,15 +12,11 @@
 from pre_commit.util import force_bytes
 
 
-class FatalError(RuntimeError):
-    pass
-
-
 def _log_and_exit(
-    msg: str,
-    ret_code: int,
-    exc: BaseException,
-    formatted: str,
+        msg: str,
+        returncode: int,
```

Comment:
Changed the name of this argument in order to consistent with the rest of the codebase, for example: https://github.com/pre-commit/pre-commit/search?q=%22returncode%22